pkg/util/exec/exec.go

```diff
@@ -251,7 +251,7 @@ func logFromReader(logger *capnslog.PackageLogger, reader io.ReadCloser) {
 	lastLine := ""
 	for in.Scan() {
 		lastLine = in.Text()
-		logger.Debug(lastLine)
```

Comment:
Where are we missing this logging? Debug logging is already activated in the OSD prepare job. This change probably just means that the operator pod will log a lot of stdout from ceph commands, which I don't think is the intent.

Reply:
Hum, it's not the prepare job, it's the OSD main container but you are right this might cause another side effect. Actually, this will print the output of all the Executor calling `ExecuteCommand`, which is relatively small. We can probably reduce this to only the OSD. What do you think?

Reply:
What if we set an env var on the OSD pod? If the env var is set, log to Info, if it is not set, log to Debug.